content/en/apps/guides/hosting/monitoring.md

```diff
@@ -0,0 +1,72 @@
+
+## Outside the CHT
+
+Be sure to monitor important items that the CHT depends on in order to be healthy. You should alert when any of these are close to their maximum(disk space)  or minimum (days left of valid TLS certificate):
+
+* Domain expiration with registrar
+* TLS certificate expiration 
+* Disk & swap space
+* CPU utilization
+* Memory utilization
+* Network utilization
+* Process count
+* OS Uptime
+
+## Inside the CHT
+
+As of [CHT 3.9.0]({{% ref "core/releases/3.9.0.md#instance-monitoring" %}}), the [Monitoring v1 API]({{< relref "apps/reference/api#get-apiv1monitoring" >}}) has been available. In  [CHT 3.12.0]({{% ref "core/releases/3.12.0.md#instance-monitoring" %}}) this API was depricated in [favor of v2]({{< relref "apps/reference/api#get-apiv3monitoring" >}}). These APIs do not require any authentication and so can easily be used with third party tools as they do not need a CHT user account.
+
+All metrics need to be monitored over time so that you can easily see longitudinal patterns when debugging an outage or slow down. The names below are extrapolated from the paths in the JSON returned by the API and should be easy to find when viewing the `/api/v1/monitoring` or `/api/v2/monitoring` URLs on your CHT instance.
+
+### Explosive Growth
+
+Many of the values in the monitoring API do not mean much in isolation. For example if an instance has 10,714,278 feedback docs, is that bad?  If it's years old and has thousands of users, then this is normal.  If it is 4 months old and has 100 users this is a dire problem! 
+
+You should monitor these metrics for unexpected growth as measured by percent change over 24 hours. Ideally this can be subjectively calculated when it is more than 5% growth than the prior day. 
+
+* Conflict Count
+* CouchDB Medic Doc Count
+* CouchDB Medic Fragmentation
+* CouchDB Sentinel Doc Count
+* CouchDB Sentinel Fragmentation
+* CouchDB Users Doc Count
+* CouchDB Users Fragmentation
+* CouchDB Users Meta Doc Count
+* Feedback Count
+* Messaging Outgoing State Due
+* Messaging Outgoing State Failed
+
+### Non-Zero Values
+
+Other values should always be zero, and you should alert when they are not. You may opt to alert only when they are non-zero for more than 24 hours:
+
+* Messaging Outgoing State Failed
+* Outbound Push Backlog
+* Sentinel Backlog
+
+### Zero or Near Zero Values
+
+Finally, these values should always be _not_ zero, and you should alert when are zero or very close to it.  You may opt to alert only when they are zero for more than 24 hours:
+
+* Date Uptime
```

Comment:
I think this section should be restructured into a single list, potentially a table with a column for "expected value" or similar.

Reply:
great thinking! fixed in 09f309d